src/RSCoin/Bank/Storage/Mintettes.hs

```diff
@@ -76,6 +83,7 @@ mkMintettesStorage =
     MintettesStorage
     { _msMintettes = []
     , _msPendingMintettes = []
+    , _msPermittedMintettes = Set.empty
```

Comment:
Better use `mempty`. It's more general and we'll have to do less work if we decide to change this type (e. g. to `HashSet`). In new code we use `mempty` in such cases.